src/cm_adapter.js

```diff
@@ -1104,13 +1104,20 @@ class CMAdapter {
 
   indentLine(line, indentRight = true) {
     const { editor } = this
-    const cursors = editor._getCursors();
+    let cursorConfig;
+    if (editor._getViewModel != null) {
```

Comment:
Remove the null check. Just checking for `editor._getViewModel` would be fine as it would be `undefined` otherwise.